src/main/scala/com/cognite/sdk/scala/v1/sessions.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1
+
+final case class Session(
+    id: Long,
+    `type`: String,
+    status: String,
+    nonce: String,
+    clientId: String
+)
+
+final case class SessionCreateWithToken(
+    tokenExchange: Boolean = true
+)
```

Comment:
Out of interest, what's this value actually for?

Reply:
I have no idea it's just mandatory on the api contract 😅 
![image](https://user-images.githubusercontent.com/92026183/139872983-6502debe-3b8a-4c55-b946-0c66c98f304b.png)